wiki/Community/Discord_servers/osu!_University/sr.md

```diff
@@ -0,0 +1,45 @@
+
+## Догађаји
+
+| Датум | Догађај | Сродни линкови |
+| :-- | :-- | :-- |
+| 2021-08-09/2021-08-13 | Недеља интервјуа са најбољим играчима (контрола стреса) | [YouTube плејлиста](https://www.youtube.com/playlist?list=PL_QhgSQeB2zCEiwKnHwliUGbtjE4hI0MG) |
+| 2021-08-30/2021-09-03 | Недеља интервјуа са најбољим играчима (mindblock) | [YouTube плејлиста](https://www.youtube.com/playlist?list=PL_QhgSQeB2zD4GdoGOympAkhhSIytKect) |
+| 2021-09-27/2021-10-01 | Недеља интервјуа са најбољим играчима (високи AR) | [YouTube плејлиста](https://www.youtube.com/playlist?list=PL_QhgSQeB2zDGmnvGdJJLGei0iLvmvtZg) |
+| 2021-10-11/2021-10-15 | Недеља интервјуа са најбољим играчима (стратегија турнира) | [YouTube плејлиста](https://www.youtube.com/playlist?list=PL_QhgSQeB2zBFW_YpeW_F_lDQRO2J9IOl) |
+| 2022-02-01/2022-03-01 | Издржљивост Тапкања | [Документ информација](https://docs.google.com/document/d/14yY4_9E6q8kbOwPGtvWG7z5MCQsQGrX168hAOZFAK_M/edit), [документ резултата](https://docs.google.com/document/d/1aJmRXZTFCR2FirXQHC5f-TTYUCimpBPWGBg-bZPb_YE/edit), [видео за дискусију о резултатима](https://www.youtube.com/watch?v=BINGyy43T4U) |
+| 2022-02-20 | [osu! састанак заједнице](/wiki/Community/osu!_community_meetings) #11 | [YouTube VOD](https://www.youtube.com/watch?v=d66pU5lsHvE), [сажетак чланка](https://osu.ppy.sh/home/news/2022-03-07-community-meetings-recap) |
+| 2022-03-06 | [osu! састанак заједнице](/wiki/Community/osu!_community_meetings) #12 | [YouTube VOD](https://www.youtube.com/watch?v=HimCHAnPCCY), [сажетак чланка](https://osu.ppy.sh/home/news/2022-03-07-community-meetings-recap) |
+| 2022-05-20/2022-05-22 | [osu! Догађај разговора: Превазилажење препрека](/wiki/Community/osu!_Talk_Event/Overcoming_Obstacles) | [YouTube плејлиста](https://www.youtube.com/playlist?list=PL_QhgSQeB2zC5_Go2esnB7_64Egv5HmSL) |
```

Comment:
```suggestion
| 2022-05-20/2022-05-22 | [osu! Talk Event: Превазилажење препрека](/wiki/Community/osu!_Talk_Event/Overcoming_Obstacles) | [YouTube плејлиста](https://www.youtube.com/playlist?list=PL_QhgSQeB2zC5_Go2esnB7_64Egv5HmSL) |
```
Since this is the name of the event as per the wiki page linked and there isn't a great way to translate it it's better like this IMO